_data/Kikemascote/Kikemascote.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Enrique Mascote 
+institution: UTCH  
+profile_pic: Kike.jpg 
+quote: Let's get this work togheter
```

Comment:
Fix typo? Change `togheter` to `together`.

Reply:
```suggestion
quote: Let's get this work together
```

done